# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1316390979 25200
#      Sun Sep 18 17:09:39 2011 -0700
# Node ID 8c7caa009a1eb42877b4d9865c594f00ccd8b6d1
# Parent  25effffba9b0b21083d25e5fbfa7a11de95b8227
__makeinfo__.m: Correct error string for third argument.

* __makeinfo__.m: Correct error string for third argument.

diff --git a/scripts/help/__makeinfo__.m b/scripts/help/__makeinfo__.m
--- a/scripts/help/__makeinfo__.m
+++ b/scripts/help/__makeinfo__.m
@@ -78,18 +78,17 @@ function [retval, status] = __makeinfo__
           ("\nSee also:", sprintf (" %s,", T{:})(1:end-1), "\n");
     else
       fsee_also = @(T) strcat ...
           ("\nSee also:", sprintf (" @ref{%s},", T{:})(1:end-1), "\n");
     endif
   endif
 
   if (! isa (fsee_also, "function_handle"))
-    error (["__makeinfo__: third input argument must ", ...
-            "be the empty matrix, or a function handle"]);
+    error ("__makeinfo__: third input argument must be a function handle");
   endif
 
 
   ## It seems like makeinfo sometimes gets angry if the first character
   ## on a line is a space, so we remove these.
   text = strrep (text, "\n ", "\n");
 
   ## Handle @seealso macro
